framework/architecture/acl/tests.rst

```diff
@@ -2,7 +2,7 @@ Tests
 =====
 
 The easiest way to test the ACL rules of a module is by extending
-:java:ref:`ch.tocco.nice2.persist.security.inject.AbstractModuleAclTest`. This base test class automatically sets the
+:abbr:`ch.tocco.nice2.persist.security.inject.AbstractModuleAclTest (AbstractModuleAclTest)`. This base test class automatically sets the
```

Comment:
:java:ref: wurden falsch ersetzt in diesem File (der volle Klassenname sollte als Popup angzeigt werden, nicht umgekehert)